### PR TITLE
feat: skeleton example and api update

### PR DESCRIPTION
**Pull request by vidhi499**

### CODE REVIEW

File: src/theme/base/typography.ts [hunk 1]
```diff
@@ -12,20 +12,20 @@ const typography = {
     '5xl': 1.5,
   },
   lineHeights: {
-    'none': 1,
```
Comment (MD-REHMAN) [inline src/theme/base/typography.ts:15]: Revert these changes

File: src/components/composites/Skeleton/SkeletonCircle.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import React, { memo, forwardRef } from 'react';
```
Comment (MD-REHMAN) [inline src/components/composites/Skeleton/SkeletonCircle.tsx:1]: Can reuse Skeleton component, instead of re-writing all logic.

File: example/storybook/stories/components/primitives/Box/basic.tsx [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
```
Comment (MD-REHMAN) [inline example/storybook/stories/components/primitives/Box/basic.tsx:1]: Revert these changes

File: src/components/composites/Skeleton/Context.tsx [hunk 4]
```diff
@@ -0,0 +1,2 @@
+import { createContext } from 'react';
```
Comment (MD-REHMAN) [inline src/components/composites/Skeleton/Context.tsx:1]: Why Skeleton have context?

File: src/components/composites/Skeleton/Skeleton.tsx [hunk 5]
```diff
@@ -7,7 +7,7 @@ import type { ISkeletonProps } from './types';
 
 const Skeleton = (allProps: ISkeletonProps, ref: any) => {
   const isDomUsable = canUseDom();
-  const { variant, children, ...props } = allProps;
+  const { children, ...props } = allProps;
   const newProps = useThemeProps('Skeleton', props);
```
Comment (MD-REHMAN) [inline src/components/composites/Skeleton/Skeleton.tsx:11]: usePropResolution should be use in place of use useThemeProps

**Review by MD-REHMAN — APPROVED** [review]

**Review by Ajamuar — APPROVED** [review]
